app/move/views/cancel.njk

```diff
@@ -18,12 +18,13 @@
 {% endblock %}
 
 {% block formActions %}
-  {{ govukWarningText({
-    html: t("moves::cancel.warning", {
-      name: move.person.fullname | upper
-    }),
-    iconFallbackText: "Warning"
-  }) }}
-
+  {% if move.status != "proposed" %}
```

Comment:
Why do we not see the warning message for proposed moves?

Reply:
I asked the same to @Andy-Millar ! 😄 It's because `proposed` moves aren't accepted by the supplier yet, so no need to notify them.